Source/WebInspectorUI/UserInterface/Views/ColorPicker.js

```diff
@@ -90,6 +90,18 @@ WI.ColorPicker = class ColorPicker extends WI.Object
         this._color = WI.Color.fromString("white");
 
         this._dontUpdateColor = false;
+        this._nodeStyles = nodeStyles || null;
+        if (this._nodeStyles && this._nodeStyles.length != 0) {
```

Comment:
this should probably be
```suggestion
        if (this._nodeStyles && this._nodeStyles.length !== 0) {
```
or
```suggestion
        if (this._nodeStyles && this._nodeStyles.length) {
```